src/summary_statistics/tmrca.cc

```diff
@@ -22,13 +22,14 @@
 #include "tmrca.h"
 
 void TMRCA::calculate(const Forest &forest) {
-  if (forest.calcSegmentLength() == 0) return;
+  //std::cout << "forest.calcSegmentLength() "<<forest.calcSegmentLength() <<std::endl;
+  if (forest.rec_bases_.size() == 1) return;
```

Comment:
Oh, I missed that change. The two `if`s do pretty different things here, don't they? Why was the original one wrong? I don't think that is fixes the uninitialized value. I'll look into it.